it/test/uk/gov/hmrc/crdlcache/repositories/CustomsOfficeListsRepositorySpec.scala

```diff
@@ -69,7 +70,7 @@ class CustomsOfficeListsRepositorySpec
 
   def withCustomsOfficeEntries(
     offices: Seq[CustomsOffice]
-  )(test: ClientSession => Future[Assertion]): Unit = {
+  )(test: ClientSession => Future[Assertion | Seq[Assertion]]): Unit = {
```

Comment:
Hmm what happens when one of these assertions fails? I can't find a mention of this signature in [asynchronous testing](https://www.scalatest.org/user_guide/async_testing) (but that doesn't mean it won't work of course!)